sources/Application/Instruments/SampleInstrument.cpp

```diff
@@ -678,6 +688,14 @@ bool SampleInstrument::Render(int channel, fixed *buffer, int size,
               fpSpeed = rp->speed_;
             }
             break;
+          case SILM_LOOP_PINGPONG:
+             if (rp->position_ <= rp->rendLoopStart_ || input <= loopPosition) {
+               rpReverse = !rpReverse;
+               fpSpeed = -fpSpeed;
+               input=loopPosition ; // OK forward loop, regular backwards loop,
+               rp->couldClick_=SHOULD_KILL_CLICKS;
```

Comment:
Doesn't seem like this is being used anywhere, unless we understand what the original intent was in implementing this variable, I don't think we should include it.

Reply:
Are you referring to rp->couldClick? Yeah I can get rid of it =)